clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerService.java

```diff
@@ -243,15 +243,15 @@ public void unsubscribeAll(PubSubTopic versionTopic) {
    * Stop specific subscription associated with the given version topic.
    */
   @Override
-  public void unSubscribe(PubSubTopic versionTopic, PubSubTopicPartition pubSubTopicPartition) {
-    PubSubConsumerAdapter consumer = getConsumerAssignedToVersionTopicPartition(versionTopic, pubSubTopicPartition);
+  public void unSubscribe(PubSubTopic versionTopic, PubSubTopicPartition pubSubTopicPartition, long timeoutMs) {
+    SharedKafkaConsumer consumer = getConsumerAssignedToVersionTopicPartition(versionTopic, pubSubTopicPartition);
```

Comment:
This change suggested by @haoxu07 is really neat. `PubSubConsumerAdapter` is under `venice-common` and cannot access the default value static variable because it's in `clients/da-vinci-client`. Thus, it cannot easily create an overloaded version of `unSubscribe()` referencing the default wait.